msgpack/msgpack.cabal

```
@@ -120,4 +120,4 @@ test-suite msgpack-tests
                   , tasty              == 1.2.*
                   , tasty-quickcheck   == 0.10.*
                   , tasty-hunit        == 0.10.*
-                  , QuickCheck         == 2.12.*
+                  , QuickCheck         >= 2.12 && < 2.14
```

For testsuite deps I prefer to support single major version ranges, i.e.

```suggestion
                  , QuickCheck         == 2.13.*
```

as otherwise we'd have to make all major versions are covered at least once in the CI; and as 

https://matrix.hackage.haskell.org/#/package/QuickCheck

shows QC 2.13 has build-plans for all GHCs